XPlat/RadialProgress/nuget/Xamarin.RadialProgress.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.RadialProgress</id>
+    <title>Xamarin Radial Progress</title>
+    <version>1.0.2</version>
+    <authors>Xamarin Inc.</authors>
+    <owners>Xamarin Inc.</owners>
```

Comment:
Either MSFT or Xamarin is fine in this case.  Because this is not a change in features, just an oss release, I am deciding to keep Xamarin.